contracts/MemberToken.sol

```diff
@@ -28,6 +28,12 @@ contract MemberToken is ERC1155Supply, Ownable {
         controllerRegistry = IControllerRegistry(_controllerRegistry);
     }
 
+    // Provides metadata value for the opensea wallet. Must be set at construct time
+    // Source: https://www.reddit.com/r/ethdev/comments/q4j5bf/contracturi_not_reflected_in_opensea/
+    function contractURI() public view returns (string memory) {
+        return "https://orcaprotocol-nft.vercel.app/assets/contract-metadata";
```

Comment:
we need to make this updateable

Reply:
that could always change in the future though we don't want to be locked in to a single server

Reply:
fair enough